api/v1alpha1/helmchartproxy_types.go

```diff
@@ -60,6 +60,104 @@ type HelmChartProxySpec struct {
 	// fields from each selected workload Cluster and programatically create and set values.
 	// +optional
 	ValuesTemplate string `json:"valuesTemplate,omitempty"`
+
+	// Options represents the helm setting options which can be used to control behaviour of helm operations(Install, Upgrade, Delete, etc)
+	// via options like wait, skipCrds, timeout, waitForJobs, etc.
+	// +optional
+	Options *HelmOptions `json:"options,omitempty"`
+}
+
+type HelmOptions struct {
+	// DisableHooks prevents hooks from running during the Helm install action.
+	// +optional
+	DisableHooks bool `json:"disableHooks,omitempty"`
+
+	// Wait enables the waiting for resources to be ready after a Helm install/upgrade has been performed.
+	// +optional
+	Wait bool `json:"wait,omitempty"`
+
+	// WaitForJobs enables waiting for jobs to complete after a Helm install/upgrade has been performed.
+	// +optional
+	WaitForJobs bool `json:"waitForJobs,omitempty"`
+
+	// DependencyUpdate indicates the Helm install/upgrade action to get missing dependencies.
+	// +optional
+	DependencyUpdate bool `json:"dependencyUpdate,omitempty"`
+
+	// Timeout is the time to wait for any individual Kubernetes operation (like
+	// resource creation, Jobs for hooks, etc.) during the performance of a Helm install action.
+	// Defaults to '10 min'.
+	// +optional
+	Timeout *metav1.Duration `json:"timeout,omitempty"`
+
+	// SkipCRDs controls whether CRDs should be installed during install/upgrade operation.
+	// If set, no CRDs will be installed.
+	// By default, CRDs are installed if not already present.
```

Comment:
Nit: maybe move this back to the line above.

Reply:
Sure, updated. Please review.